common_benchmark_suite/openxla/benchmark/unique_ids.py

```diff
@@ -56,6 +56,34 @@
 OUTPUT_DATA_BERT_LARGE_FP16_JAX_384X1024XF16 = f"{OUTPUT_DATA_BERT_LARGE_JAX_384X1024}-fp16"
 OUTPUT_DATA_BERT_LARGE_BF16_JAX_384X1024XBF16 = f"{OUTPUT_DATA_BERT_LARGE_JAX_384X1024}-bf16"
 
+################################################################################
+# ResNet50 models                                                              #
+################################################################################
+MODEL_IMPL_RESNET50 = f"aff75509-4420-40a8-844e-dbfc48494fe6-MODEL_IMPL"
+MODEL_RESNET50 = "aff75509-4420-40a8-844e-dbfc48494fe6-MODEL_RESNET50"
+MODEL_RESNET50_FP32 = f"{MODEL_RESNET50}-fp32"
+MODEL_RESNET50_FP16 = f"{MODEL_RESNET50}-fp16"
+MODEL_RESNET50_BF16 = f"{MODEL_RESNET50}-bf16"
+
+# JAX ResNet50 mdoels
+MODEL_IMPL_RESNET50_JAX = f"{MODEL_IMPL_RESNET50}-JAX"
+
+MODEL_RESNET50_FP32_JAX = f"{MODEL_RESNET50_FP32}-JAX"
+MODEL_RESNET50_FP32_JAX_3X224X224XF32 = f"{MODEL_RESNET50_FP32_JAX}-3x224x224xf32"
+
+MODEL_RESNET50_FP16_JAX = f"{MODEL_RESNET50_FP16}-JAX"
+MODEL_RESNET50_FP16_JAX_3X224X224XF16 = f"{MODEL_RESNET50_FP16_JAX}-3x224x224xf16"
+
+MODEL_RESNET50_BF16_JAX = f"{MODEL_RESNET50_BF16}-JAX"
+MODEL_RESNET50_BF16_JAX_3X224X224XBF16 = f"{MODEL_RESNET50_BF16_JAX}-3x224x224xbf16"
+
+INPUT_DATA_RESNET50_FP32_JAX_3X224X224XF32 = "b2676fcd-1cc1-4b79-84a2-71c7efaeb58b"
```

Comment:
Should these have the `-fp32`, `-fp16` and `-bf16` suffixes?

Reply:
Done.